azure-pipelines.yml

```diff
@@ -0,0 +1,83 @@
+# File: azure-pipelines.yml
+
+trigger:
+  batch: true
+  branches:
+    include:
+      - main
+      - release/*
+
+resources:
+  repositories:
+    - repository: niveristand-custom-device-build-tools
+      type: github
+      ref: main
+      endpoint: ni (2)
+      name: ni/niveristand-custom-device-build-tools
+# TEMPORARY, include fake custom device to use templates until migrating the templates into build tools makes sense
+    - repository: imitation-cd
+      type: github
+      ref: main
+      endpoint: papowerNI
+      name: papowerNI/imitation-cd
+
+pool:
+  name: custom-device-temp
+  demands:
+    - agent.os -equals Windows_NT
+
+# --------- Build Parameters here ---------
+parameters:
+# Custom Device Name
+  - name: customDevice
+    type: string
+    default: niveristand-slsc-eds-custom-device
+# LabVIEW Versions to Build
+  - name: lvVersions
+    type: object
+    default:
+      - '2020'
+      - '2021'
+      - '2023'
+# Custom Device Build Steps
+  - name: buildSteps
```

Comment:
`buildSpecs` perhaps? No strong preference right now, but I can imagine a more complex custom device might want to run custom steps which are not build specs (and wouldn't be placed here).

Reply:
I avoided "buildSpecs" since the object may contain more than just one build spec on a line, and I didn't want the "each" statement that says "each buildSpec in buildSpecs" to be incorrect.  If the object type of buildSteps changes, I could see this being modified as well.

Reply:
I had originally missed that this is only used in this file, and not by the common build tools. This makes sense.